core/src/components/cat-tag/cat-tag.tsx

```diff
@@ -160,25 +167,19 @@ export class CatTag {
   }
 
   @Watch('errors')
-  onErrorsChanged(value?: boolean | string[] | ErrorMap) {
+  onErrorsChanged(newValue?: boolean | string[] | ErrorMap, _oldValue?: unknown, update: boolean = true) {
```

Comment:
But it's called everywhere in only one matter:

```this.onErrorsChanged(this.errors, undefined, false);```

Why then other params needed, when will they be used?

Reply:
Its a little bit hacky. The method is called via the @Watch annotation with the new and old value (and the update value missing, thus defaulting to true). So this kind of keeps the old logic, but it can also be called explicitly once a component loads (but without the update).